composeApp/src/commonMain/kotlin/domain/Event.kt

```diff
@@ -0,0 +1,19 @@
+package domain
+
+data class Event (
+    val id: String,
+    val title: String,
+    val description: String,
+    val createdAt: String,
+    val lastUpdate: String,
+    val eventBannerUrl : String,
+    val speakers: Speaker,
+    val evenType: EventType,
+    val eventLocation:EventLocation,
+    val addressInfo:String
+)
+enum class EventType{METTUP,WORKSHOP}
```

Comment:
Same here, it would be better if we have `EventType{ METTUP, WORKSHOP }`

Reply:
done